solnlib/log.py

```diff
@@ -79,7 +79,7 @@ class Logs(metaclass=Singleton):
     _default_directory = None
     _default_namespace = None
     _default_log_format = (
-        "%(asctime)s %(levelname)s pid=%(process)d tid=%(threadName)s "
+        "%(asctime)s log_level=%(levelname)s pid=%(process)d tid=%(threadName)s "
```

Comment:
Can we add a test case for the default log format? I think these test cases do not exist, but we can start adding as and when the changes happen.